### PR TITLE
#152799113 setup database and ORM and install some necessary dependencies

### PR DESCRIPTION
**Pull request by chinemelu**
#### What does this PR do?
setup postgresql and sequelize 

#### Description of Task to be completed?
Install and setup the following dependencies
body-parser v1.18.2
 dotenv v4.0.0
 express v4.16.2
 morgan v1.9.0
 pg v7.4.0
 pg-hstore v2.3.2
 sequelize v4.22.8
 babel-cli v6.26.0
 babel-preset-env v1.6.1
 babel-register v6.26.0
 eslint v4.11.0
 eslint-config-airbnb v6.1.0
 nodemon  v1.12.1

configure and setup sequelize and postgres to the point that models and migration fiels can be created successfully

#### What are the relevant pivotal tracker stories?
#152799113

### CODE REVIEW

**Comment by chinemelu** [comment]
@andela-uibrahim, please help me review my PR

File: app.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const dotenv = require('dotenv').config();
+
+const app = express();
+
+app.use(logger('dev'));
+app.use(bodyparser.json());
+app.use(bodyparser.urlencoded({ extended: true }));
+
+const port = process.env.PORT || 3000;
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply app.js:15]: you may want to configure your hound ci to use eslint configurations

File: app.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+app.set('port', port);
+
+const server = app.listen(port, () => {
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply app.js:19]: you may want to configure your hound ci to use eslint configurations

File: app.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  console.log('listening to port 3000');
+});
+
+
+export { app, server };
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply app.js:24]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:1]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+const path = require('path');
+const Sequelize = require('sequelize');
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:3]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:5]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+const config = require(`${__dirname}/../config/config.json`)[env];
+const db = {};
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:7]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+let sequelize;
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:9]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  );
+}
+
+fs
+  .readdirSync(__dirname)
+  .filter(file =>
+    (file.indexOf('.') !== 0) &&
+    (file !== basename) &&
+    (file.slice(-3) === '.js'))
+  .forEach(file => {
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:24]: you may want to configure your hound ci to use eslint configurations

File: server/models/index.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    const model = sequelize.import(path.join(__dirname, file));
+    db[model.name] = model;
+  });
+
+Object.keys(db).forEach(modelName => {
```
[the remark this answers is not in the document]
Comment (freemile2ru) [reply server/models/index.js:29]: you may want to configure your hound ci to use eslint configurations

**Comment by freemile2ru** [comment]
for this PR, I don't like the content of your description of tasks completed

**Review by freemile2ru — COMMENTED** [review]
looks good to me. Just fix the hound ci problems

**Pull request merged**